### PR TITLE
Improve the tutorial that may be confusing

### PR DESCRIPTION
**Pull request by hyukmyeong**
This patch updates the tutorial which can be confusing for users
This issue has been raised by the users from

https://github.com/google/googletest/issues/2950

I tried minimize the revision because I don't know exactly what the previous writer intended
and so almost all of the original contents have been preserved

### CODE REVIEW

File: googletest/README.md [hunk 1]
```diff
@@ -24,17 +24,20 @@ another project.
 When building Google Test as a standalone project, the typical workflow starts
 with:
 
-    mkdir mybuild       # Create a directory to hold the build output.
-    cd mybuild
-    cmake ${GTEST_DIR}  # Generate native build scripts.
+    git clone https://github.com/google/googletest.git -b release-1.10.0
```
Comment (romanwerpachowski) [inline googletest/README.md:27]: I would add here something like "Even if you want to build only one of the components (e.g. just `googletest`), you have to create your build directory in the main directory of the cloned repo. For people who are as thick as me :)

**Review by romanwerpachowski — APPROVED** [review]
Thanks!

File: googletest/README.md (hunk 1 above)
Comment (hyukmyeong) [reply googletest/README.md:27]: I hope this additional revision would reflect what you commented
Please let me know if there are more parts that need to be corrected

**Review by romanwerpachowski — COMMENTED** [review]
Thanks, that's a great improvement.

**Comment by vslashg** [comment]
This looks good, but could you revert the changes adding a space between "GoogleTest" and "GoogleMock"?  I know we're inconsistent with this spelling in places, but we prefer to spell these as one word.

**Comment by hyukmyeong** [comment]
@vslashg 

As I modified `README.md` in googletest directory, I couldn't avoid to modify other two `README.md`s for consistency.
(even though there are still some inconsistencies)
It is very cautious to revise the main title, but for now, I uploaded a patch for it.
Please let me know if there are some parts which can't be modified

**Comment by Celthi** [comment]
This PR breaks a bunch of links such as the "gmock for dummies" in https://github.com/google/googletest/blob/master/googlemock/README.md

**Pull request merged**